.changes/unreleased/Docs-20220328-094152.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Docs
```

Comment:
@jtcohen6 do we want `Docs` to just be documentation changes generally or strictly for `dbt-docs` changes?  

Apart from this PR - it might be worth updating the changie README with more details on each change type (ie, be sure to add details of what will break on a breaking change).

Reply:
just my opinions:
- original intent with `Docs` was "strictly for dbt-docs changes"
- generally I think a change like this can be `Skip changelog` — README, reflected in GitHub only, no real change to the functionality of installed `dbt-core`

Reply:
I was going for a changelog entry for all the things!  But it makes sense that this is a good candidate for `Skip Changelog`.  @McKnight-42 can you make that happen?